code-review-chat/CodeReviewChat.ts

```diff
@@ -141,7 +141,13 @@ export class CodeReviewChat extends Chatter {
 					}),
 				]);
 
-				const hasExisting = existingReviews?.data?.length || existingRequests?.data?.users?.length;
+				// Check if has existing reviews made ignoring PR author since comments they leave count as reviews
+				const hasExistingReview = existingReviews.data.some((review) => {
```

Comment:
I'd keep in the null checks, GH likes `null`.

Reply:
Haha sorry I missed that Copilot generated this snipped